internal/module/services/version_manager/version_manager_test.go

```diff
@@ -240,3 +243,68 @@ func Test_PublishVersion(t *testing.T) {
 		}
 	})
 }
+
+// Test_ListModuleVersions checks:
+// - if correct response is returned when versions are fetched
+func Test_ListModuleVersions(t *testing.T) {
+	t.Parallel()
+
+	t.Run("Listing versions", func(t *testing.T) {
+		db := &mocks.DynamoDB{
+			ScanOut: &dynamodb.ScanOutput{
+				Items: []map[string]types.AttributeValue{
+					{
+						"Version": &types.AttributeValueMemberS{Value: "1.0.1"},
+					},
+					{
+						"Version": &types.AttributeValueMemberS{Value: "1.0.10"},
+					},
+					{
+						"Version": &types.AttributeValueMemberS{Value: "1.0.2-beta"},
+					},
+					{
+						"Version": &types.AttributeValueMemberS{Value: "1.0.2-alpha"},
+					},
+					{
+						"Version": &types.AttributeValueMemberS{Value: "v1.0.2-gamma"},
+					},
+					{
+						"Version": &types.AttributeValueMemberS{Value: "0.0.0"},
+					},
+					{
+						"Version": &types.AttributeValueMemberS{Value: "0.0.0-alpha"},
+					},
+				},
+			},
+		}
+
+		svc := &VersionManagerService{Db: db}
+
+		req := services.ListModuleVersionsRequest{Module: "dummy"}
+		res, err := svc.ListModuleVersions(context.TODO(), &req)
+
+		if err != nil {
+			t.Errorf("Expected no error, got %v", err)
+		}
+		if res.Versions[0] != "0.0.0-alpha" {
+			t.Errorf("Expected version 0.0.0-alpha, got %v", res.Versions[0])
+		}
+		if res.Versions[1] != "0.0.0" {
+			t.Errorf("Expected version 0.0.0, got %v", res.Versions[1])
+		}
+		if res.Versions[2] != "1.0.1" {
+			t.Errorf("Expected version 1.0.1, got %v", res.Versions[2])
+		}
+		if res.Versions[3] != "1.0.2-alpha" {
+			t.Errorf("Expected version 1.0.2-alpha, got %v", res.Versions[3])
+		}
+		if res.Versions[4] != "1.0.2-beta" {
+			t.Errorf("Expected version 1.0.2-beta, got %v", res.Versions[4])
+		}
+		if res.Versions[5] != "1.0.10" {
+			t.Errorf("Expected version 1.0.10, got %v", res.Versions[5])
+		}
```

Comment:
Can you refactor this into an array of expected results and then use a reflect.DeepEqual to do the comparison.
```suggestion
                expectedVersions = []string {
                "0.0.0-alpha",
                "0.0.0",
                "1.0.1",
                "1.0.2-alpha",
                "1.0.2-beta",
                "1.0.10",                
                }
                if !reflect.DeepEqual(res.Versions, expectedVersions) {
                    t.Errorf("Versions do not match, got %v, want %v", res.Versions, expectedVersions)
                }
```